app/src/main/kotlin/com/mapzen/erasermap/view/TimeView.kt

```diff
@@ -11,13 +11,20 @@ import com.mapzen.erasermap.R
 public class TimeView(context: Context, attrs: AttributeSet) : TextView(context, attrs) {
     public var timeInMinutes: Int = 0
         set (value) {
-            if (value < 1) {
+            if (noTime) {
+                text = ""
+            } else if (value < 1) {
                 text = context.getString(R.string.less_than_one_minute)
             } else {
                 formatTime(value)
             }
         }
 
+    public var noTime: Boolean = false
```

Comment:
NOTE: Did not know how to handle "time = 0" case when no route was found, hence this `noTime` thingy!.